### PR TITLE
Tweaks for Mike taking over for Miguel

### PR DESCRIPTION
**Pull request by mkruskal-google**
- Update names
- Spelling/grammar fixes
- Reworded a bit to match my voice

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -182,7 +182,7 @@ message Person {
 
 ``` [2]
 # github.com/protocolbuffers/protoscope
-1: {"Miguel Young de la Sota"}
+1: {"Matthew Kulukundis"}
```
Comment (fowles) [inline index.md]: `Matthew Fowles Kulukundis` according to the binary fragment ;)

File: index.md [hunk 2]
```diff
@@ -207,8 +207,8 @@ message Person {
 
 ``` [2,3]
 # github.com/protocolbuffers/protoscope
-1: {"Miguel Young de la Sota"}
-2: {"Cambridge, MA"}
+1: {"Matthew Kulukundis"}
```
Comment (fowles) [inline index.md]: here too

File: index.md [hunk 3]
```diff
@@ -233,8 +233,8 @@ message Person {
...
```
Comment (fowles) [inline index.md]: and here

**Review by fowles — APPROVED** [review]

File: index.md (hunk 1 above)
Comment (mkruskal-google) [reply index.md]: Whoops :P

**Pull request merged**